public/instances.json

```
@@ -1,4 +1,33 @@
 {
+  "P2B Compliance": {
+    "languages": ["*eu"],
+    "jurisdictions": ["EU"],
+    "stats": {
+      "services": "208",
+      "documents": "253"
+    },
+    "industries": {
+      "en": "Online intermediation services for businesses subject to the European <a href=\"https://eur-lex.europa.eu/eli/reg/2019/1150/oj\">platforms-to-businesses (“P2B” / 2019/1150) regulation</a>",
+      "fr": "Services d’intermédiation en ligne pour les entreprises sujets au règlement européen <a href=\"https://eur-lex.europa.eu/legal-content/FR/TXT/HTML/?uri=CELEX:32019R1150&from=EN\">P2B / 2019/1150</a>"
+    },
+    "maintainers": [
+      {
+        "name": "Ambassadeur français pour le numérique",
+        "logo": "/images/contributors/ambnum.png",
+        "url": "https://disinfo.quaidorsay.fr/fr"
+      },
+      {
+        "name": "Open Evidence",
+        "logo": "/images/contributors/open-evidence.png",
+        "url": "https://open-evidence.com/"
+      },
+      {
+        "name": "European Comission",
+        "logo": "/images/contributors/european-commission.jpg",
+        "url": "https://europa.eu"
```

Let's make that DGCNECT.

```suggestion
        "url": "https://ec.europa.eu/info/departments/communications-networks-content-and-technology_en"
```